dbus_lynx_distributor/dbus_lynx_distributor_service.py

```diff
@@ -84,19 +80,19 @@ def _config_getboolean(self, option, fallback):
         return self._config.getboolean(f'ftdi:{self._ftdi.serial_number}', option, fallback=fallback)
 
     def _update(self):
+        sleep(5)
         try:
-            for lynx in range(4):
+            for lynx in range(3, -1, -1):
```

Comment:
What's the reason for the changed order?

Reply:
because the lynx are hanging upside down on my wall - I guess it would be good to make this a configurable option